cmd/server/flags.go

```diff
@@ -72,6 +72,16 @@ var flags = []cli.Flag{
 		Name:    "server-key",
 		Usage:   "server ssl key path",
 	},
+	&cli.StringFlag{
```

Comment:
I wonder if we could use env vars with value too instead of require a file ...

Reply:
Transporting CSS or JS code via a ENV VAR would IMHO require the content to be BASE64 encoded, to avoid special character issues. And that would be cumbersome.

I'm not sure about the need to do so, since e.g. in a Kubernetes environment, one can mount custom CSS and JS files from a ConfigMap, see https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#populate-a-volume-with-data-stored-in-a-configmap.
This trick works easily with large CSS or JS files as well (not just the single-line data in the documented example).

For any Woodpecker users, provisioning non-Kubernetes systems, I would assume some mechanics for provisioning custom files are present as well (thinking of puppet, chef, ansible).

This makes me wonder, in which scenario Woodpecker users/admins would require the ability to provision custom CSS and JS as ENV VAR without file mapping?

Reply:
well that's a good argument against my idea yes